docs/build-intercept.rst

```diff
@@ -183,36 +194,36 @@ Then to manually build UnifyFS:
     $ make
     $ make install
 
-.. note:: **Fortran Compatibility**
-
-    To build with gfortran compatibility, include the ``--enable-fortran``
-    configure option:
+To see all available build configuration options, type ``./configure --help``
+after ``./autogen.sh`` has been run.
 
-    ``./configure --prefix=/path/to/install/ --enable-fortran``
+---------------------------
 
-    There is a known `ifort_issue <https://github.com/LLNL/UnifyFS/issues/300>`_
-    with the Intel Fortran compiler as well as an `xlf_issue <://github.com/LLNL/UnifyFS/issues/304>`_
-    with the IBM Fortran compiler. Other Fortran compilers are currently
-    unknown.
+-------------------------------
+Building UnifyFS with Autotools
+-------------------------------
 
-To see all available build configuration options, type ``./configure --help``
-after ``./autogen.sh`` has been run.
+Download the latest UnifyFS release from the `Releases
+<https://github.com/LLNL/UnifyFS/releases>`_ page or clone the develop branch
+from the `UnifyFS repository <https://github.com/LLNL/UnifyFS>`_.
 
-.. TODO: Add a section in build docs that shows all the build config options
+Build the Dependencies
+**********************
 
-Build the Dependencies without Spack
-"""""""""""""""""""""""""""""""""""""
+UnifyFS requires MPI(in progress of removing), LevelDB, GOTCHA, FlatCC, Margo
```

Comment:
Until we no longer require MPI, we should just say we depend on it. Telling the users that we are in the process of removing that requirement doesn't really help them.

Reply:
Removed the parenthetical.